docs/data/people.yml

```diff
@@ -686,3 +686,18 @@
     - cloud computing
     - containers
     - water resources
+
+- first_name: Micah
+  last_name: Wengren
+  institution: NOAA
+  website: https://www.google.com/search?q=micah+wengren
```

Comment:
Welcome Micah! Do you intend to list your website as a google search for your name?  This field is optional, and if you don't have a website, it's fine to leave it blank.

Reply:
Well, that was sort of tongue in cheek, but actually a google search of my name happens to get the job done as far as a CV goes, so it's a stand in for a page I have to maintain myself :-)